angular-template.js

```diff
@@ -7,5 +7,28 @@ module.exports = {
             Overrides to Angular template recommended rules:
             https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin-template/src/configs/recommended.json
         */
+
+        /*
+            Overrides to Angular rules outside of the recommended configuration:
+        */
+
+        '@angular-eslint/template/accessibility-alt-text': 'error',
```

Comment:
This requires that the [img, area, object, and input elements](https://github.com/angular-eslint/angular-eslint/blob/c8d2321181576152d307fc9fd4c836fdc28915fb/packages/eslint-plugin-template/src/rules/accessibility-alt-text.ts#L33) have alt text.

I'm not aware of any projects that currently have ARIA accessibility requirements, and projects that don't wouldn't to enforce them. If that is the common case, then I would propose leaving these rules disabled even if they don't result in errors in preliminary testing. I've left a comment with notes following testing each rule, and given those results, I would propose the following unless this is not actually common.

**Proposal**
Leave the following rules disabled.
- `@angular-eslint/template/accessibility-alt-text`
- `@angular-eslint/template/accessibility-elements-content`
- `@angular-eslint/template/accessibility-label-for`
- `@angular-eslint/template/accessibility-label-has-associated-control`

**Question**
[template-accessibility-table-scope](http://codelyzer.com/rules/template-accessibility-table-scope/) and [template-accessibility-valid-aria](http://codelyzer.com/rules/template-accessibility-valid-aria/) validate correct usage of ARIA attributes. Do we leave these disabled, because we don't expect their use?

Reply:
We discussed our philosophy around accessibility in our meeting today. I have an action item to reach out to some software leaders to discuss where it fits in our priority list and whether shared libraries should allow for accessible apps or or do what they can to require accessible apps.

Reply:
Comment copied from below because I forgot this thread had already started:
We discussed the accessibility rules in our meeting today. Our decision was to disable all of them for now in our recommended config. This (sadly) most accurately reflects where the company is today with accessibility--it's not something that developers should spend time enforcing.

There is an upcoming push from the Design team and the design system to start to care more. At that time we can revisit this decision and hopefully turn the rules on with the backing of leadership to spend developer time making apps accessible.

Trever will update the PR to reflect this with a comment something like this:
"At the moment, there's no NI-wide requirement to enforce accessible apps. These rules can be useful if individual apps choose to prioritize it. They may be enabled in the future as the priority of accessibility increases."